test/e2e/next-test/next-test.test.ts

```diff
@@ -0,0 +1,196 @@
+import {
+  cpSync,
+  mkdtempSync,
+  readFileSync,
+  rmSync,
+  statSync,
+  writeFileSync,
+} from 'fs'
+import { tmpdir } from 'os'
+import { join } from 'path'
+
+import { nextTest } from 'next-test-utils'
+import { FileRef, nextTestSetup } from 'e2e-utils'
+import { spawnSync } from 'child_process'
+
+function createTemporaryFixture(fixtureName: string) {
+  const fixturePath = join(__dirname, fixtureName)
+  const stat = statSync(fixturePath)
+  if (!stat.isDirectory()) {
+    throw new Error(`Fixture ${fixtureName} is not a directory.`)
+  }
+
+  const dir = mkdtempSync(join(tmpdir(), fixtureName))
+  cpSync(fixturePath, dir, { recursive: true })
+  return dir
+}
+
+describe('next test', () => {
+  const { next: basicExample } = nextTestSetup({
+    files: new FileRef(join(__dirname, 'basic-example')),
+    dependencies: {
+      '@playwright/test': '1.43.1',
+    },
+    skipStart: true,
+  })
+
+  afterAll(async () => {
+    await basicExample.destroy()
+  })
+
+  describe('first time setup', () => {
+    it.each([['first-time-setup-js'], ['first-time-setup-ts']])(
+      'should correctly install missing dependencies and generate missing configuration file for %s',
+      async (fixtureName: string) => {
+        const fixture = createTemporaryFixture(fixtureName)
+
+        try {
+          const { stdout, stderr } = await nextTest(fixture, [], {
+            stderr: true,
+            stdout: true,
+            cwd: fixture,
+          })
+
+          expect(stdout).toContain('Installing devDependencies')
+          expect(stdout).toContain('@playwright/test')
+          expect(stdout).toContain(
+            `Successfully generated ${
+              fixtureName === 'first-time-setup-js'
+                ? 'playwright.config.js'
+                : 'playwright.config.ts'
+            }. Create your first test and then run \`next experimental-test\`.`
+          )
+
+          expect(stderr).toBe('')
+
+          const pkgJSON = JSON.parse(
+            readFileSync(join(fixture, 'package.json'), 'utf-8')
+          )
+
+          expect(pkgJSON.devDependencies).toHaveProperty('@playwright/test')
+        } finally {
+          rmSync(fixture, { recursive: true, force: true })
+        }
+      }
+    )
+
+    it('should fail if next.js config is missing experimental#testProxy', async () => {
+      const fixture = createTemporaryFixture('first-time-setup-js')
+
+      try {
+        writeFileSync(join(fixture, 'next.config.js'), 'module.exports = {}')
+        const { stdout, stderr } = await nextTest(fixture, [], {
+          stderr: true,
+          stdout: true,
+          cwd: fixture,
+        })
+
+        expect(stdout).toBe('')
```

Comment:
I think you'll need to remove this check as it's failing when running the tests in PPR (since that logs a warning)

Reply:
probably makes sense to just remove them then